core/src/main/java/org/elasticsearch/common/unit/ByteSizeUnit.java

```diff
@@ -225,6 +234,23 @@ public long toPB(long size) {
 
     static final long MAX = Long.MAX_VALUE;
 
+    private final byte id;
```

Comment:
Sure. I usually don't like to use ordinal to serialize enums but I agree, we're not going to add new units anytime soon :)